modules/serverless-logic-linking-workflows-eventing-system.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-workflow-eventing-system
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-linking-workflows-eventing-system_{context}"]
+= Linking workflows to the eventing system
+
+The {ServerlessLogicOperatorName} links workflows with the eventing system using Knative Eventing, SinkBindings, and triggers. These objects are created automatically by the {ServerlessLogicOperatorName} and simplify the production and consumption of workflow events.
+
+The following example shows the Knative Eventing objects created for an `example-workflow` configured with a platform-scoped eventing system:
+
+.Example of platform-scoped eventing system configuration
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+  namespace: example-namespace
+spec:
+  eventing:
+    broker:
+      ref:
+        name: example-broker <1>
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+  services:
+    dataIndex: <2>
+      enabled: true
+    jobService: <3>
+      enabled: true
+  ...
+----
+
+<1> The `example-broker` is used by the Data Index, Jobs Service, and the `example-workflow`.
+<2> The Data Index is deployed with ephemeral configurations.
+<3> The Jobs Service is deployed with ephemeral configurations.
+
+The `example-broker` is a Kafka class Broker, and its configuration is defined in the `kafka-broker-config` ConfigMap.
+
+The following example displays how to configure a Kafka Knative Broker for use with the SonataFlowPlatform:
+
+.Example of Knative Kafka Broker used by the SonataFlowPlatform
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: Kafka <1>
+  name: example-broker
+  namespace: example-namespace
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+----
+
+<1> The Kafka class is used to create the `example-broker`.
```

Comment:
```suggestion
<1> The Kafka class is used to create `example-broker`.
```

or the `example-broker` object.

Reply:
I will use: _The Kafka class is used to create the `example-broker` object._